lib/license_finder/package_managers/bundler_package.rb

```diff
@@ -8,6 +8,7 @@ def initialize(spec, bundler_def, options={})
         spec.name,
         spec.version.to_s,
         options.merge(
+          authors: spec.authors.to_s.tr('\/[\/]', ''),
```

Comment:
BTW, `join` might have fixed your original problems formatting authors... That is, `to_s` on an array adds a bunch of characters that are difficult to `tr` away.